src/components/ContactDetails.tsx

```diff
@@ -46,19 +46,36 @@ const ContactDetails = () => {
         <>
             <h1>{targetContact.name}</h1>
             <address>
-                gender: {targetContact.gender} <br />
-                birthday: {typeof targetContact.birthday === 'number' ? (new Date(targetContact.birthday)).toLocaleDateString() : null}<br />
-                email: <a href="mailto:">{targetContact.email}</a><br />
-                phone: <a href={`tel:+${targetContact.phone.country_code}${targetContact.phone.phone}`}>{`${targetContact.phone.country_code}${targetContact.phone.phone}`}</a><br />
-                address: <br />
-                {targetContact.address.street} <br />
-                {targetContact.address.unit} <br />
-                {targetContact.address.city}, {targetContact.address.state} <br />
-                {targetContact.address.zip} <br />
-                {targetContact.address.country}
-            </address> 
-            <Link to={`/contacts/${contactID}/update`} state={contactID}>Update Contact</Link>
-            <DeleteContactButton id={contactID ? contactID : ""}/>
+                <div>
+                    <h4>gender: </h4>
+                    <p>{targetContact.gender}</p>
+                </div>
+                <div>
+                    <h4>birthday: </h4>
+                    <p>{typeof targetContact.birthday === 'number' ? (new Date(targetContact.birthday)).toLocaleDateString() : null}</p>
```

Comment:
`time` is a more semantic element here, and more accessible for screen readers I believe

```suggestion
                    <time datetime={new Date(targetContact.birthday)). toIso8601String()}>{typeof targetContact.birthday === 'number' ? (new Date(targetContact.birthday)).toLocaleDateString() : null}</time>
```

Reply:
Also, no need for ternary when the false outcome is null
```suggestion
                    <p>{typeof targetContact.birthday === 'number' && (new Date(targetContact.birthday)).toLocaleDateString()}</p>
```

Reply:
Just checking that I understand this right? Would `new Date(targetContact.birthday)).toLocaleDateString()` return `null` if `targetContact.birthday` had not been filled out by the user?

Reply:
If the user doesn't fill out a date in the input then `targetContact.birthday' is an empty string, with threw an error in the Date constructor. I found a way to fix it with `&&` though. The code seems to be working now.